File: java/jimmer-cloud/store-service/src/main/java/org/babyfish/jimmer/example/cloud/store/StoreServiceApp.java
package org.babyfish.jimmer.example.cloud.store;

import org.babyfish.jimmer.client.EnableImplicitApi;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@EnableImplicitApi
@SpringBootApplication
public class StoreServiceApp {

    public static void main(String[] args) {
        SpringApplication.run(StoreServiceApp.class, args);
    }
}

File: kotlin/jimmer-cloud-kt/store-service/src/main/kotlin/org/babyfish/jimmer/example/cloud/kt/store/StoreServiceApp.kt
package org.babyfish.jimmer.example.cloud.kt.store

import org.babyfish.jimmer.client.EnableImplicitApi
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@EnableImplicitApi
@SpringBootApplication
class StoreServiceApp

fun main(args: Array<String>) {
    runApplication<StoreServiceApp>(*args)
}